zephyr/include/sof/schedule/task.h

```diff
@@ -79,8 +78,6 @@ struct pipeline_task {
 
 static inline enum task_state task_run(struct task *task)
 {
-	assert(task->ops.run);
-
```

Comment:
@finikorg wouldn't be better to make assert a NULL operation on zehpyr? Thus you don't need to change this code.

Reply:
@dbaluta I am thinking adding to zephyr/include assert.h making use of Zephyr assert() functions.

Reply:
We should still add a check here that takes the role of the current non-disable-able assert. Otherwise we would just send IP to NULL which isn't exactly good. Possibly panic if this isn't supposed to be NULL (and I think it's a valid response here)